defaults/helpers.go

```diff
@@ -18,3 +33,188 @@ func ociProfileConversionCheck(profile secprofile.Profile, entitlementID string)
 
 	return ociProfile, nil
 }
+
+// FIXME factor for all defaults package and even project-wide
+func testSpec() *specs.Spec {
+	s := &specs.Spec{
+		Process: specs.Process{
+			Capabilities: &specs.LinuxCapabilities{
+				Bounding:    []string{},
+				Effective:   []string{},
+				Inheritable: []string{},
+				Permitted:   []string{},
+				Ambient:     []string{},
+			},
+		},
+		Linux: &specs.Linux{
+			Seccomp:   &specs.LinuxSeccomp{},
+			Resources: &specs.LinuxResources{},
+			IntelRdt:  &specs.LinuxIntelRdt{},
+		},
+		Windows: &specs.Windows{
+			Resources: &specs.WindowsResources{
+				Memory:  &specs.WindowsMemoryResources{},
+				CPU:     &specs.WindowsCPUResources{},
+				Storage: &specs.WindowsStorageResources{},
+				Network: &specs.WindowsNetworkResources{},
+			},
+		},
+	}
+
+	seccomp, err := setDefaultSeccompProfile()
+	if err != nil {
+		// In case we get an error before seccomp struct is fully updated from decoded json, we empty it manually.
+		s.Linux.Seccomp = &specs.LinuxSeccomp{}
+	}
+
+	s.Linux.Seccomp = seccomp
+
+	return s
+}
+
+// We can't create a map to check set inclusion with specs.LinuxSeccompArg as key type in Golang so no O(n)
+func seccompArgsContains(syscallArgsSet, syscallArgsSubset []specs.LinuxSeccompArg) bool {
```

Comment:
if this is only used for testing, we should consider moving it to the test file

Reply:
totally, actually wanted to do that in the first place 😖 will fix